pkgs/servers/web-apps/discourse/default.nix

```diff
@@ -34,7 +34,7 @@ let
     gawk
 
     # Image optimization
-    imagemagick
+    imagemagickBig # imagemagickBig is used to get get ghostscript fonts, needed for local letter avatars
```

Comment:
I think we should overwrite the imagemagick input.

Reply:
Thanks and done, @SuperSandro2000